src/index.js

```diff
@@ -147,6 +147,9 @@ class LoggerPlugin {
       if (!this.logs.length) {
         return true;
       }
+      if (typeof this.invocationInstance.context.iopipe.label === 'function') {
```

Comment:
@pselle This happens after the `logs.length` check, which means there's logs. Confused by your review.

Reply:
Ah, what I mean is, it should only label when there are logs successfully attached to an invocation, that is, that the upload was successful. With this, if the upload fails, it will say there are logs but there won't be any if that upload fails.